packages/babel-plugin-transform-react-jsx/test/fixtures/spread-transform/transform-to-object-assign/output.js

```diff
@@ -1,3 +1,5 @@
-var div = /*#__PURE__*/React.createElement(Component, Object.assign({}, props, {
+var _reactJsxRuntime = require("react/jsx-runtime");
+
+var div = /*#__PURE__*/_reactJsxRuntime.jsx(Component, Object.assign({}, props, {
```

Comment:
It doesn't matter to keep this test as classic or not right?

Reply:
I think it's ok to use the automatic runtime, since it still tests that the `transform-to-object-assign` works on the JSX transform output.